source/octf/trace/parser/ParsedIoTraceEventHandler.cpp

```diff
@@ -152,42 +153,62 @@ class ParsedIoTraceEventHandler::Map : public NonCopyable {
 struct ParsedIoTraceEventHandler::FileId {
```

Comment:
Suggestion: use globally FileId. Probably you could create additional dir in OCTF sources and put there IFileSystemViewer as well.